src/ext/default.h

```diff
@@ -15,4 +15,9 @@
 
 int DefaultExtensionInit(RSExtensionCtx *ctx);
 
+typedef enum {
```

Comment:
not needed, see my comment above

Reply:
After discussion with @MeirShpilraien, we have decided to use it as flags to allow for future expansions.

Reply:
no longer needed